### DISCUSSION TITLE
File Upload Failure

### DISCUSSION THREAD

**Discussion by reramjiawan** [danny-avila/LibreChat#3743] [Troubleshooting]
### What happened?

When uploading a file (without setting up any settings for RAG). The file upload will obviously fail. However you are then completely locked out of chatting. Is this by intention or does the error thrown "freeze" the application? 

### Steps to Reproduce

1. Upload file
2. Observe error in log
3. Chatting is disabled

### Relevant log output

```shell
rag_api           | 2024-08-22 08:35:20,152 - root - INFO - Request POST http://rag_api:8000/embed - 200
LibreChat         | 2024-08-22 08:35:20 error: Error embedding file File embedding failed.
LibreChat         | 2024-08-22 08:35:20 error: [/files] Error processing file: File embedding failed.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
If nothing is configured, sure, you should expect errors. Can you demonstrate what you mean by freezing? Because you should be able to refresh/navigate away. We should set up a way to cancel the file download, though

**Reply by reramjiawan** [reply]
Hi Danny,

First of all thanks for the fast reply and the great project!

What I meant is that you cannot continue the chat anymore once it has failed uploading. The send button will remain disabled and you have to either refresh or start a new chat. I hope that provides enough information.

Kind regards.

**Reply by danny-avila** [reply]
Thanks! If file upload fails, the UX is expected to be immediate, there is no "Freezing"

https://github.com/user-attachments/assets/d7802234-e0dc-4599-8998-f5d4c7fa1a96

**Reply by reramjiawan** [reply]
Exactly as you see in the video you cannot then proceed your chat anymore as the send button is "locked out". I apologize for using the word freezing because that is not the case. The application keeps responding.

**Reply by danny-avila** [reply]
I can still proceed with the chat

https://github.com/user-attachments/assets/4eea97e0-c0b9-4471-b0f3-e1c6fed7ba45

**Reply by reramjiawan** [reply]
That is so weird I am using Azure OpenAI and I cannot continue my chat after the file upload failed. :(. 

https://github.com/user-attachments/assets/222c6607-8d02-4b0a-b587-1be65cd1696b

**Reply by reramjiawan** [reply]
Correction starting a new chat and sending messages is also not possible.


https://github.com/user-attachments/assets/eeb08920-8627-45d8-b7c8-7a8166332be8

**Reply by reramjiawan** [reply]
@danny-avila Hi Danny, I am sure you are very busy with the new features coming. Have you by chance seen my videos? Kind regards.

**Reply by WinoutB** [reply]
I have the same issue unfortunately
